schema/cassandra/temporal/schema.cql

```diff
@@ -94,6 +94,20 @@ CREATE TABLE tasks (
     'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'
   };
 
+-- Stores task queue information such as user provided versioning data
+CREATE TABLE task_queue_user_data (
+  namespace_id    uuid,
+  task_queue_name text,
+  data            binary, -- temporal.server.api.persistence.v1.TaskQueueUserData as proto binary blob
```

Comment:
add `data_encoding` and change the above comment to not say it's a proto

Reply:
Did this for the sql tables and forgot cassandra, thanks!